openlibrary/plugins/openlibrary/sentry.py

```diff
@@ -12,12 +13,18 @@ def is_enabled():
     return hasattr(infogami.config, 'sentry') and infogami.config.sentry.enabled
 
 
+def get_software_version():  # -> str:
```

Comment:
This function already exists here:

https://github.com/internetarchive/openlibrary/blob/44138d8fadcf86f3226e07657b7bbe1a234ae059/openlibrary/plugins/openlibrary/status.py#L28-L29

Let's reference it to keep things DRY.

Reply:
DONE